css/css.css

```
@@ -47,28 +48,36 @@ a{
 }
 
 .nav-header {
-  display: flex;
-  justify-content: space-around;
+  margin-top: 58px;
+  position: relative;
+  margin-left: 118px;
 }
 
 header {
   background-color: var(--basic-header);
-  background-size: 170px;
+  background-size: 180px;
   font-size: 16px;
-  text-transform: uppercase;
-  font-weight: 500px;
-  line-height: 30px;
   color: black;
   list-style: none;
   display: grid;
-  height: 114px;
+  height: 115px;
   grid-template-columns: min-content 1fr  min-content;
   align-items: flex-end;
   width: 1440px;
+
+  }
+
+  .header-list {
```

Лишние отступы в начале селектора. Их не должно быть. Нужно соблюдать единообразие, что бы код выглядел одинаково во всех однотипных случаях. Тогда в нем можно быстро и легко разобраться.
```suggestion
.header-list {
```